src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.scss

```diff
@@ -0,0 +1,30 @@
+@import 'src/main/webapp/content/scss/artemis-variables';
+
+jhi-collapsable-assessment-instructions {
+    max-height: 100vh;
+}
+
+.square {
+    width: 10px;
+    height: 10px;
+}
+
+.success-background {
+    background-color: $success;
+}
+
+.danger-background {
+    background-color: $danger;
+}
+
+.primary {
+    color: $primary;
+}
+
+.success {
+    color: $success;
+}
+
+.danger {
+    color: $danger;
+}
```

Comment:
Any reason these need to be redeclared? They are equivalent to the existing classes `bg-success`, `bg-danger`, `text-primary`, `text-success` and `text-danger` respectively.

Reply:
I was not aware of them @joschahenningsen. Thank you for pointing them out! Done ✅